### PR TITLE
add count to search result and add test

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###

- > add count to search result 
- > add test for empty result
- > use existing util method for formatting

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/SearchResult.java [hunk 1]
```diff
@@ -6,10 +6,14 @@
...
 
+    @JsonProperty("count")
+    public final int count;
+
     @JsonProperty("data")
     public final List<?> data;
 
     public SearchResult(List<?> data) {
         this.data = data;
+        count = (data == null) ? 0 : data.size();
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/SearchResult.java:17]: can use `CollectionUtils.size` which will handle even null :) but up to you

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/SearchResult.java:17]: which jar is it? I think it is not there.

Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/SearchResult.java:17]: ok then let's leave it as it is

File: src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/controllers/StaleBlobControllerTest.java [hunk 2]
```diff
@@ -82,6 +84,21 @@ void should_return_list_of_stale_blobs_when_there_is_by_default_param_value() th
 
     }
 
+    @Test
+    void should_return_empty_data_when_there_is_no_stale_blob() throws Exception {
+
+        given(staleBlobFinder.findStaleBlobs(2)).willReturn(null);
```
Comment (rkondratowicz) [inline src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/controllers/StaleBlobControllerTest.java:90]: shouldn't it return an empty list?

File: src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/SearchResult.java (hunk 1 above)
Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/SearchResult.java:17]: what buggers me more is that List<?> data can be null flushed

p.s. i accidentally pressed `unresolve` button

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/SearchResult.java:17]: normally `data` can not be null

File: src/main/java/uk/gov/hmcts/reform/blobrouter/util/TimeUtils.java [hunk 3]
```diff
@@ -14,6 +14,8 @@ private TimeUtils() {
     }
 
     public static String toLocalTimeZone(Instant instant) {
-        return dateTimeFormatter.format(ZonedDateTime.ofInstant(instant, EUROPE_LONDON_ZONE_ID));
+        return instant == null
+            ? ""
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/util/TimeUtils.java:18]: why not null?

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/util/TimeUtils.java:18]: envelope end point may return null

File: src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/SearchResult.java (hunk 1 above)
Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/SearchResult.java:17]: so do we then need to check for nulls? it's not normal as you pointed out

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/SearchResult.java:17]: I prefer to check.
